src/main/java/nextstep/subway/line/application/PathService.java

```diff
@@ -15,31 +21,93 @@
 import java.util.List;
 import java.util.Optional;
 
+import static nextstep.subway.fee.domain.AgePriceStrategy.calculatePrice;
+import static nextstep.subway.fee.domain.StationFeeStrategy.STANDARD_FEE;
+
 @Service
 @Transactional
 public class PathService {
+
     private StationRepository stationRepository;
     private SectionRepository sectionRepository;
+    private StationFeeRepository stationFeeRepository;
+    private LineRepository lineRepository;
 
-    public PathService(StationRepository stationRepository, SectionRepository sectionRepository) {
+    public PathService(StationRepository stationRepository, SectionRepository sectionRepository
+            , StationFeeRepository stationFeeRepository, LineRepository lineRepository) {
         this.stationRepository = stationRepository;
         this.sectionRepository = sectionRepository;
+        this.stationFeeRepository = stationFeeRepository;
+        this.lineRepository = lineRepository;
+    }
+
+    @Transactional(readOnly = true)
+    public PathResponse getStationPathInfo(final String sourceStationId, final String targetStationid, LoginMember loginMember) {
+        int totalDistance = 0;
+        int totalFee = STANDARD_FEE;
+        List<StationResponse> shortestPath = getShortestPath(sourceStationId, targetStationid);
```

Comment:
```suggestion
        PathResult pathResult = getShortestPath(sourceStationId, targetStationid);
```

메소드 안에 여러 변수들과 로직들이 썩여 있는것 같아요.
책임별로 클래스를 나눈다면 더 좋을것 같아요.